docs/localized/RU/README.md

```diff
@@ -0,0 +1,160 @@
+<img src="https://github.com/Unity-Technologies/ml-agents/blob/master/docs/images/image-banner.png" align="middle" width="3000"/>
+
+# Unity ML-Agents Toolkit v.1.0
```

Comment:
can you change this to "Unity ML-Agents Toolkit Version Release 7"

Reply:
Done.